litmus-portal/frontend/src/components/RecentWorkFlow/PassedVsFailed/styles.ts

```diff
@@ -0,0 +1,63 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: theme.spacing(40),
+    height: theme.spacing(22),
+    marginRight: theme.spacing(3),
+    borderRadius: 3,
+  },
+  boxMain: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  headerMain: {
+    fontWeight: 'bold',
+    fontSize: theme.spacing(2),
```

Comment:
font size doesn't fall under spacing....either use rem directly or a better option is to use the typography component for consistency

Reply:
@arkajyotiMukherjee  

- [x] Fixed